packages/components/src/components/CodeBlock/components/CodeBlockCode.tsx

```
@@ -0,0 +1,79 @@
+  return (
+    <>
+      <pre ref={preRef} className={s.pre}>
+        <code
+          className={clsx(
+            'hljs',
+            s.code,
+            utilClasses.typography['mono-codeblock']
+          )}
+          data-language={isHighlighted ? language : undefined}
+          // `highlight.js` escapes the raw source before wrapping tokens in spans — see useHighlightedCode.
+          dangerouslySetInnerHTML={isHighlighted ? { __html: html } : undefined}
```

Biome (2.5.8)

[error] Avoid passing both children and the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtmlWithChildren)